internal/pkg/api/error.go

```diff
@@ -481,6 +481,16 @@ func NewHTTPErrResp(err error) HTTPErrResp {
 				zerolog.InfoLevel,
 			},
 		},
+		// audit unenroll
+		{
+			ErrAuditUnenrollReason,
+			HTTPErrResp{
+				http.StatusConflict,
+				"ErrAuditReasonConflict",
+				"agent document contaiins audit_unenroll_reason",
```

Comment:
yes; though i may remove this as a 409 response as @intxgo has suggested that it may be useful to allow Endpoint to keep pinging the API so the `updated_at` timestamp gets updated